src/GafferModule/AnimationBinding.cpp

```diff
@@ -162,11 +162,11 @@ void setTieMode( Animation::Key &k, const Animation::Tangent::TieMode mode )
 std::string keyRepr( const Animation::Key &k )
 {
 	return boost::str( boost::format(
-			"Gaffer.Animation.Key( Gaffer.Animation.Time( %d ), %.9g, Gaffer.Animation.Type.%s, "
+			"Gaffer.Animation.Key( Gaffer.Animation.Time( %d ), %.9g, Gaffer.Animation.Interpolation.%s, "
```

Comment:
This poses some problems for the compatibility of serialised `.gfr` files. We try to guarantee that files saved in later versions will be loadable in earlier versions provided that new features of the later version are not used. This is an important part of the upgrade path - people need to know they can work in `0.61` for testing, but safely roll back to `0.60` if they hit problems. The rename to Interpolation is logical though. I think either we a) drop it for now, because it's not integral to getting the cubic modes out, or b) merge it separately, along with a companion PR for 0.60 which adds a monkey-patching config file with `Gaffer.Animation.Interpolation = Gaffer.Animation.Type`.

Reply:
not sure if this is waht you mean by monkey patching, but I could keep a constructor in the python api that accepts Gaffer.Animation.Type and calls the new c++ signature

Reply:
Created new PRs for option b.
https://github.com/GafferHQ/gaffer/pull/4361
https://github.com/GafferHQ/gaffer/pull/4362